mutatex/utils.py

```diff
@@ -435,12 +439,22 @@ def get_foldx_sequence(pdb, multimers=True):
                     residue_list.append(tuple(["%s%s%d" % (res_code, chain.get_id(), residue.get_id()[1])]))
                 else:
                     sequences[chain_name] += res_code
+                    positions[chain_name] += f"{residue.get_id()[1]},"
 
     if multimers:
         collated_chains = []
         seq_ids, seqs = list(zip(*list(iteritems(sequences))))
+        pos_ids, pos = list(zip(*list(iteritems(positions))))
         seq_ids = np.array(seq_ids)
+        pos_ids = np.array(pos_ids)
+        
         unique_seqs, unique_idxs = np.unique(seqs, return_inverse=True)
+        unique_pos, unique_idxp = np.unique(pos, return_inverse=True)
+        
+        if (unique_idxs == unique_idxp).all() == False:
+            log.warning("Input amino acid sequence and input position sequence is not identical in multimer.")
+            residue_list.append("RESIDUE_MISMATCH")
```

Comment:
can we raise an exception instead of returning a flag value? It's most Pythonic way to handle this type of problems. For instance this would be probably fitting for a `ValueError`